self-host/traefik/simple-example/docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
```

Comment:
What about `self-host/examples/traefik/docker-compose.yml` instead?

Reply:
Moved it